Rakefile

```diff
@@ -19,39 +18,41 @@ end
 
 desc "Show version"
 task :version do
-  puts "debug_inspector version: #{DebugInspector::VERSION}"
+  require "rubyvm/debug_inspector/version"
```

Comment:
It has a reason, that's explained [here](https://github.com/banister/debug_inspector/pull/19#issuecomment-323953234).